<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QTabWidget" name="tabSetting">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="FontSett">
        <attribute name="title">
         <string>Style &amp;&amp; Fonts</string>
        </attribute>
        <widget class="QFrame" name="fontFrame">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>6</y>
           <width>386</width>
           <height>276</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>16</number>
          </property>
          <item row="1" column="3">
           <widget class="QSpinBox" name="offsetSpinBox">
            <property name="toolTip">
             <string>How far should be displayed symbol from baseline.</string>
            </property>
            <property name="value">
             <number>17</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="balloonsLabel">
            <property name="text">
             <string>Number of balloons:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="fontImageLbl">
            <property name="text">
             <string>Image:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QSpinBox" name="ballonsSpinBox">
            <property name="toolTip">
             <string>How many &quot;balloons&quot; should be shown on image.</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>13</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="fontImageLabel">
            <property name="text">
             <string>Consolas, 12 pt</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="fontImageButton">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Change...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLabel" name="offsetLabel">
            <property name="text">
             <string>Image font offset:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="4">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>fontImageLabel</zorder>
         <zorder>fontImageLbl</zorder>
         <zorder>fontImageButton</zorder>
         <zorder>offsetLabel</zorder>
         <zorder>offsetSpinBox</zorder>
         <zorder>balloonsLabel</zorder>
         <zorder>ballonsSpinBox</zorder>
        </widget>
       </widget>
       <widget class="QWidget" name="ColorsSett">
        <attribute name="title">
         <string>Colors</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>391</width>
           <height>270</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFrame" name="colorsFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="imageFontColorLabel">
               <property name="text">
                <string>Image font:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="imageFontColorButton">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>26</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="colorRectLabel">
               <property name="text">
                <string>Selection rectangle:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>25</width>
                 <height>12</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="colorRectButton">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>26</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="colorBoxLabel">
               <property name="text">
                <string>Selection fill:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>25</width>
                 <height>12</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="rectFillColorButton">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>26</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="colorBoxLabel_2">
               <property name="text">
                <string>Boxes:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>25</width>
                 <height>12</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="3" column="2">
              <widget class="QPushButton" name="colorBoxButton">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>26</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="backgroundColorLabel">
               <property name="text">
                <string>Background:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>25</width>
                 <height>12</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="2">
              <widget class="QPushButton" name="backgroundColorButton">
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>26</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>88</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="TextSett">
        <attribute name="title">
         <string>Text settings</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>183</width>
           <height>256</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_1">
            <item>
             <widget class="QLabel" name="lblWordSpace">
              <property name="text">
               <string>Word space:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="sbWordSpace">
              <property name="toolTip">
               <string>If the space between 2 symbol will be bigger than this number, space will be inserted between these symbols</string>
              </property>
              <property name="value">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lblParaIndent">
              <property name="text">
               <string>Paragraph indent:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="spParaIndent">
              <property name="toolTip">
               <string>If the space of  first letter in line is bigger than this number, new paragraph will be started</string>
              </property>
              <property name="value">
               <number>15</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblLigatures">
              <property name="text">
               <string>Ligatures:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="pteLigatures">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="plainText">
               <string>fi
fl</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QCheckBox" name="cbOpenDialog">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>381</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Open dialog before import/export</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="lblLigaturesHelp">
         <property name="geometry">
          <rect>
           <x>165</x>
           <y>90</y>
           <width>221</width>
           <height>171</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-size:8pt;&quot; style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Ligatures&lt;/span&gt; are used during import of text file to boxes/symbols.&lt;/li&gt;
&lt;li style=&quot; font-size:8pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Enter &lt;span style=&quot; font-weight:600;&quot;&gt;one&lt;/span&gt; &lt;span style=&quot; font-weight:600;&quot;&gt;ligature&lt;/span&gt; per line&lt;/li&gt;
&lt;li style=&quot; font-size:8pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Spaces&lt;/span&gt; will be keeped&lt;/li&gt;
&lt;li style=&quot; font-size:8pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Empty lines&lt;/span&gt; will be removed&lt;/li&gt;
&lt;li style=&quot; font-size:8pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Duplicates&lt;/span&gt; will be removed too&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>195</x>
           <y>10</y>
           <width>186</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;b&gt;Word space&lt;/b&gt; and &lt;b&gt;Paragraph indent&lt;/b&gt; it used during export to text file</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="TesseractSett">
        <attribute name="title">
         <string>Tesseract</string>
        </attribute>
        <widget class="QLabel" name="lblPrefix">
         <property name="geometry">
          <rect>
           <x>1</x>
           <y>20</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string notr="true">TESSDATA_PREFIX:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="pbSelectDP">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>20</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Select...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lnPrefix">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>20</y>
           <width>192</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lblLang">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>60</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Language:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="cbLang">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>60</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Available languages&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pbCheck">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>60</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Check</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblNote">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>371</width>
           <height>91</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;Note:&lt;/span&gt; You must set &lt;span style=&quot; font-weight:600;&quot;&gt;Data Path&lt;/span&gt; with tesseract (&amp;gt;=3.00 and &amp;lt;=3.02) language files! Otherwise program can not find language data files. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
